api/src/test/resources/io/strimzi/api/kafka/model/044-Crd-kafkauser.yaml

```diff
@@ -162,6 +162,13 @@ spec:
                   description: A quota on the maximum bytes per-second that each client
                     group can fetch from a broker before the clients in the group
                     are throttled. Defined on a per-broker basis.
+                controllerMutationRate:
+                  type: double
+                  minimum: 0
+                  description: ' The quota on rate at which mutations are accepted
```

Comment:
As above I would get rid of the `quota on` part, unless it means something specific?